code/modules/reagents/reagent_containers/hypospray.dm

```diff
@@ -26,7 +27,7 @@
 
 ///Handles all injection checks, injection and logging.
 /obj/item/reagent_containers/hypospray/proc/inject(mob/living/affected_mob, mob/user)
-	if(!reagents.total_volume)
+	if(used_up)
 		to_chat(user, span_warning("[src] is empty!"))
```

Comment:
`to_chat(user, span_warning("[src] tip is broken and is now unusable!"))`

Explains in more detailed on what happened.

Reply:
Alright